dev/fattest.simplicity/src/componenttest/rules/repeater/EERepeatTests.java

```diff
@@ -10,18 +10,27 @@
  *******************************************************************************/
 package componenttest.rules.repeater;
 
-/**
- *
- */
 public class EERepeatTests {
 
     public static enum EEVersion {
         EE7, EE8, EE9, EE7_FULL, EE8_FULL, EE9_FULL
     }
 
-    public static FeatureReplacementAction getEEAction(EEVersion version, String serverName, String clientName) {
-        FeatureReplacementAction action = null;
-        switch (version) {
+    /**
```

Comment:
AFAIK we remove unsupported features from a repeated server configuration by adding builder method `.removeFeature("<unsupported-feature>")` to the appropriate repeat action.  I think @aguibert should have a look at this, just in case I've misinterpreted the simplicity changes.